_posts/2016/2016-04-12-hr-quality-assurance.md

```diff
@@ -0,0 +1,135 @@
+
+Due to the poor conditions in the labour market, I was looking for a job outside of Ukraine.
+I checked [Linkedin](https://www.linkedin.com/) for job ads, as it is an international platform and the information there is in English.
+I came across the ePages job ad for a Quality Assurance Engineer and I was able to apply for the job without knowing a word of German.
+
+### Why Germany?
+
+If you want to move from Ukraine to somewhere else, the most famous countries are USA and Germany.
+And Germany of course is nearer to Ukraine than the USA.
+In my view, Germany is the best country to find a job.
+
+### How did the application process work?
+
+I applied for the job in December 2014 and then was invited for a number of interviews via Skype.
+The first one was with the Technical Recruiter.
+Then I had two more technical interviews: one with developers in Jena and another with Team Ocean in Hamburg, for which this job actually was.
+Finally, I talked to Harm Behrens, the ePages CTO.
+He offered me to come to Hamburg for a trial working day.
+
+### All your interviews have been very successful. What happened then?
+
+Of course I travelled to Hamburg to do my trial working day.
+I got to know Team Ocean and I was shown their workflows and specialities.
+Furthermore, I got a very good impression of the office and my future colleagues.
+I noticed the familiar atmosphere at once.
+All in all, I had a very good impression of ePages and I wanted that job.
+And what shall I say... here I am.
+I'm now a member of Team Ocean.
+
+###  How did you manage your relocation?
+
+Right after I received the OK for the job at ePages, I applied for the VISA docs.
+End of August I moved to Hamburg, where ePages provided me with a preliminary apartment.
+I immediately started to look for my own apartment to relocate also my wife and my daughter.
+
+### Was it hard to find an apartment?
+
+Sometimes it was a bit hard because of all the documents that are required to rent an apartment.
+But the team helped me a lot with the communication.
+Right in October, I started German lessons.
+I attend German classes now twice a week.
+
+### Did you get support with other administrative issues?
+
+Well, German was nowhere really required.
+Bank account, health insurance, registration... all that worked fine just with English.
+
+### Tell us about your first weeks at ePages.
+
+Well, there's not much to say: I was integrated from the very first day.
+It really felt like I became part of another family.
+People were so nice and everybody helped me.
+
+After just one week with ePages, we had the ePages YOU company event.
+It was nice to meet all the other colleagues.
+You know that when you start a new job: everyone knows about you, but you don't know the colleagues.
+I also made first contact there with Team Ocean's twin team in Barcelona.
+
+### And the technical aspects?
+
+I opened a GitHub account the day I started with the team, and already did my first commit the next day.
```

Comment:
Probably I was not right about first commit the next day. My github account shows first pushed things in 2 weeks. :) I think it's because I could not really push it before we finish with PayPal Plus.